plugins/uefi/fu-plugin-uefi.c

```diff
@@ -413,10 +413,7 @@ fu_plugin_uefi_load_config (FuPlugin *plugin, FuDevice *device, GError **error)
 	fu_device_set_metadata_integer (device, "RequireESPFreeSpace", sz_reqd);
 
 	/* shim used for SB or not? */
-	require_shim_for_sb = fu_plugin_get_config_value (plugin, "RequireShimForSecureBoot");
-	if (require_shim_for_sb == NULL ||
-	    g_ascii_strcasecmp (require_shim_for_sb, "true") == 0)
-		shim_needed = TRUE;
+	shim_needed = fu_plugin_get_config_value_boolean (plugin, "RequireShimForSecureBoot");
```

Comment:
I think this (unintentionally) inverted the default for this value.  If it's not set, or invalid it will be false - but previously it was true if not set or invalid.

Reply:
I think that's fine; I think we should standardize on the key being `FALSE` if missing or invalid -- returning true for a commented out key was a bit weird.

Reply:
I don't really like that if someone deletes an unmodified `uefi.conf` the behavior changes.  Perhaps we should rename the key then?

Reply:
>  Perhaps we should rename the key then

That might be best indeed; something like `DisableShimForSecureBoot`?

Reply:
Yes, that's good.

Reply:
Something like this?